mozdownload/parser.py

```diff
@@ -89,5 +89,5 @@ def handle_data(self, data):
         if not self.active_url:
             return
 
-        if self.active_url == data:
+        if data in self.active_url:
```

Comment:
Wouldn't it be safer to strip the final slash and then compare? `data in self.active_url` is a very generous condition.